packages/contracts-bedrock/src/L1/ProtocolVersions.sol

```diff
@@ -37,8 +37,8 @@ contract ProtocolVersions is OwnableUpgradeable, ISemver {
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.1-beta.1
-    string public constant version = "1.0.1-beta.1";
```

Comment:
We can undo this semver bump too

Reply:
why?

Reply:
resolving this to get it merged.

Reply:
Because we didn't make any changes to this contract in this PR

Reply:
Spoke offline and agreed we did not need to bump the semver here since no changes to this contract were made. 

However, we decided to leave this no-op version bump instead of fixing it: Because the contract was already on `beta.1`, fixing this with a follow up PR means it's semver goes from `beta.1` -> `beta.2` -> `beta.1` on develop, breaking the invariant that a semver should only increase. This might cause issues with tooling that looks at semvers, and the increment is harmless in this case since it's just a beta increment